app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,39 @@
+<nav class="navbar  is-primary">
+      <div class="navbar-brand">
+        <%=link_to root_path, class: "navbar-item" do%>
+          <h1 class="title is-5 has-text-white">Private Event</h1>
+        <% end %>
+        <div class="navbar-burger burger" data-target="navbarExample">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </div>
+      <div id="navbarExample" class="navbar-menu">
+        <div class="navbar-end">
+          <div class="navbar-item">
+            <div class="field is-grouped">
+              
+              <% if user_signed_in? %>
+                <p class="control">
+                  <%= link_to 'New Event', new_event_path, class: "button is-primary "  %>
+                </p>
+              <p class="control">
+                <%=link_to current_user.email, user_event_path(current_user), class: "button is-info is-inverted"%>
+              </p>
+              <p>
+                <%= link_to "Log out", destroy_user_session_path, method: :delete, class:"button is-info" %>
+              </p>
+            <% else %>
+            <p class="control">
+              <%= link_to 'Sign In', new_user_session_path, class: "button is-primary" %>
+            </p>
+						<p class="control">
+              <%= link_to 'Sign Up', new_user_registration_path, class: "button is-primary" %>
+            </p>
+            <% end %>
```

Comment:
Move this logic to a helper. Views are used to render information that became available to them from the corresponding controllers and not for generating it.